src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1309,17 +1325,20 @@ private bool AddOrganelle(OrganelleTemplate organelle)
     private bool MoveOrganelle(OrganelleTemplate organelle, Hex oldLocation, Hex newLocation, int oldRotation,
         int newRotation)
     {
+        // TODO: consider allowing rotation inplace
```

Comment:
Could you link to the relevant github issue here?

Reply:
I meant in the source code. You can either link in this TODO to the issue or you can remove this comment that was added in this PR.